### ISSUE TITLE
Enable Datadog PHP Tracer breaks Symfony 4.3 application

### ISSUE THREAD

**Issue by annuh** [DataDog/dd-trace-php#462]
When I enable the Datadog PHP Tracer, I'm see the following error logs:
```
[Tue Jun 04 10:18:50.091199 2019] [php7:error] [pid 15790] [client 81.204.130.234:10120] PHP Fatal error:  Uncaught ErrorException: Catchable Fatal Error: Object of class Symfony\\Component\\HttpKernel\\Event\\ExceptionEvent could not be converted to string in /opt/datadog-php/dd-trace-sources/src/DDTrace/Integrations/Symfony/V4/SymfonyBundle.php:147

Stack trace:
#0 /vendor/symfony/http-kernel/HttpKernel.php(222): Symfony\\Component\\EventDispatcher\\EventDispatcher->dd_trace_callback(Object(Symfony\\Component\\HttpKernel\\Event\\ExceptionEvent), 'kernel.exceptio...')
#1 [internal function]: Symfony\\Component\\HttpKernel\\HttpKernel->handleException(Object(ErrorException), Object(Symfony\\Component\\HttpFoundation\\Request), 1)
#2 /opt/datadog-php/dd-trace-sources/src/DDTrace/Integrations/Symfony/V4/SymfonyBundle.php(124): dd_trace_forward_call()
#3 /vendor/symfony/http-kernel/HttpKernel.php(79): Symfony\\Component\\HttpKernel\\HttpKernel->dd_trace_callback(Object(ErrorException), Object(Symfony\\Component\\HttpFoundation\\Request), 1)
#4 [internal function]: Sym in /opt/datadog-php/dd-trace-sources/src/DDTrace/Integrations/Symfony/V4/SymfonyBundle.php on line 147<br />
```

Any ideas what to do?

We are using:
PHP: PHP 7.3.6-1+ubuntu16.04.1+deb.sury.org+1 (cli) (built: May 31 2019 11:06:26) ( NTS )
Datadog Agent: v6.11.2
Datadog PHP tracer: 0.26.0
Symfony: v4.3.0

I installed the tracer using:

```
curl -L https://github.com/DataDog/dd-trace-php/releases/download/0.26.0/datadog-php-tracer_0.26.0_amd64.deb --output datadog-php-tracer.deb
dpkg -i datadog-php-tracer.deb
sudo service apache2 restart
```

**Comment by mglinski** [comment]
See: #461

**Comment by SammyK** [comment]
Thanks for the report @annuh! This issue was patched in #466. The [latest tracer version](https://github.com/DataDog/dd-trace-php/releases/tag/0.27.2) contains a patch for this issue. Let us know if the issue persists after upgrading! :)

**Comment by annuh** [comment]
Yes it's fixed, thanks! 😃
